Gemfile.lock

```diff
@@ -4,7 +4,7 @@ PATH
     cardboard_cms (0.5.0)
       bootstrap-datepicker-rails
       bootstrap-sass (~> 3.2)
-      bootstrap-wysihtml5-rails (~> 0.3.2.100)
+      bootstrap-wysihtml5-rails (>= 0.3.2.100)
```

Comment:
i think this is not a good idea as newer versions might break how we use this gem. How about `~> 0.3.2`